docs/feature_macros.md

```diff
@@ -4,7 +4,107 @@ Macros allow you to send multiple keystrokes when pressing just one key. QMK has
 
 !> **Security Note**: While it is possible to use macros to send passwords, credit card numbers, and other sensitive information it is a supremely bad idea to do so. Anyone who gets a hold of your keyboard will be able to access that information by opening a text editor.
 
-## `SEND_STRING()` & `process_record_user`
+## Using Macros In JSON Keymaps
+
+You can define up to 32 macros in a `keymap.json` file, as used by [Configurator](newbs_building_firmware_configurator.md), `qmk compile`, and XAP. You can define these macros in a list under the `macros` keyword, like this:
```

Comment:
```suggestion
You can define up to 32 macros in a `keymap.json` file, as used by [Configurator](newbs_building_firmware_configurator.md), and `qmk compile`. You can define these macros in a list under the `macros` keyword, like this:
```

Reply:
Is there a reason we're excluding XAP here? One of my intentions is to have all 3 aligned when it comes to macros.

Reply:
XAP is not currently a thing. Can be amended when it's merged.